e2e/browser/test/fixtures.ts

```diff
@@ -0,0 +1,159 @@
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import { test as base } from "@inrupt/internal-playwright-helpers";
+
+import { getNodeTestingEnvironment } from "@inrupt/internal-test-env";
+import {
+  getSourceUrl,
+  saveFileInContainer,
+  getPodUrlAll,
+  createContainerInContainer,
+} from "@inrupt/solid-client";
+
+import { Session } from "@inrupt/solid-client-authn-node";
+import issueAccessRequest from "../../../src/gConsent/request/issueAccessRequest";
+
+export { expect } from "@inrupt/internal-playwright-helpers";
+
+export type Fixtures = {
+  accessRequest: string;
+  idp: string;
+};
+const saveTextFile = async (options: {
+  name: string;
+  contents: string;
+  containerUrl: string;
+  session: Session;
+}): Promise<string> => {
+  const data = Buffer.from(options.contents, "utf-8");
+
+  const file = await saveFileInContainer(options.containerUrl, data, {
+    contentType: "application/json",
+    fetch: options.session.fetch,
+  });
+
+  const url = getSourceUrl(file);
+
+  return url;
+};
+
+const createAccessRequest = async (
+  id,
+  secret,
+  issuer,
+  vcProvider,
+  owner,
+  resource
+) => {
+  const session = new Session();
+
+  await session.login({
+    clientId: id,
+    clientSecret: secret,
+    oidcIssuer: issuer,
+  });
+
+  try {
+    const accessRequest = await issueAccessRequest(
+      {
+        access: { read: true },
+        purpose: [
+          "https://w3c.github.io/dpv/dpv/#UserInterfacePersonalisation",
+          "https://w3c.github.io/dpv/dpv/#OptimiseUserInterface",
+        ],
+        resourceOwner: owner,
+        resources: [resource],
+      },
+      {
+        fetch: session.fetch,
+        accessEndpoint: vcProvider,
+      }
+    );
+    return accessRequest;
+  } catch (error) {
+    console.log(error);
+    return error;
+  }
+};
+
+export const test = base.extend<Fixtures>({
+  // playwright expects the first argument to be a destructuring pattern.
+  // eslint-disable-next-line no-empty-pattern
+  accessRequest: async ({}, use) => {
+    const ownerSession = new Session();
+    const setupEnvironment = getNodeTestingEnvironment();
+    try {
+      await ownerSession.login({
+        oidcIssuer: setupEnvironment.idp,
+        clientId: setupEnvironment.clientCredentials.owner.id,
+        clientSecret: setupEnvironment.clientCredentials.owner.secret,
+        // DPoP is the default, but prints a warning on node.js that pollutes
+        // the test output; we can do what we need with Bearer tokens.
+        tokenType: "Bearer",
+      });
+    } catch (err) {
+      throw new Error(`Failed to login: ${(err as Error).message}`);
+    }
+
+    if (!ownerSession.info.isLoggedIn || !ownerSession.info.webId) {
+      throw new Error("Failed to login when creating test container");
+    }
```

Comment:
Can we use [`getAuthenticatedSession`](https://github.com/inrupt/typescript-sdk-tools/blob/9feabf6cd4b6058f2418c0c920123c71c6d9260f/packages/internal-test-env/index.ts#L337) from our internal tooling here instead please.